### PR TITLE
pgv: bump dependency to 30da78c.

### PR DESCRIPTION
**Pull request by htuch**
This should unblock Windows, fix the long CLI issue and also fuzz weak link problems.

Signed-off-by: Harvey Tuch <htuch@google.com>

### CODE REVIEW

**Comment by htuch** [comment]
@sesmith177 @rodaine

File: api/bazel/repositories.bzl [hunk 1]
```diff
@@ -1,13 +1,20 @@
+BAZEL_SKYLIB_SHA = "b5f6abe419da897b7901f90cbab08af958b97a8f3575b0d3dd062ac7ce78541f"
```
Comment (moderation) [inline api/bazel/repositories.bzl]: In order to be consistent with other dependencies what about:
```
BAZEL_SKYLIB_RELEASE = "0.5.0"
BAZEL_SKYLIB_SHA = "b5f6abe419da897b7901f90cbab08af958b97a8f3575b0d3dd062ac7ce78541f"
```

**Comment by moderation** [comment]
FYI, with this PR I've been able to upgrade to `rules_go` 0.14.1 which aligns with PGV's version which is currently at 0.14.0 - https://github.com/lyft/protoc-gen-validate/blob/master/WORKSPACE#L7. Great to be able to move past 0.11.2

**Comment by sesmith177** [comment]
It looks like we'll need one more set of changes to PGV to get it to work on Windows; we've submitted https://github.com/lyft/protoc-gen-validate/pull/104

The issue is the `DELETE` enum here: https://github.com/envoyproxy/envoy/blob/0c4c00e16a2905c34f94dbd81b3c5978aa4d7e01/api/envoy/api/v2/core/base.proto#L128

Specifically, `<windows.h>` defines a `DELETE` macro, so if it ever gets included before the generated `base.pb.h`, the enum will be overwritten and compilation will fail.

**Comment by htuch** [comment]
@sesmith177 ack. Are we good to merge this PR as an intermediate step?

**Comment by sesmith177** [comment]
yep, we'll PR the update to PGV. 

It also looks like this comment is no longer true now:

https://github.com/envoyproxy/envoy/blob/0c4c00e16a2905c34f94dbd81b3c5978aa4d7e01/api/bazel/api_build_system.bzl#L127-L130

**Comment by moderation** [comment]
@sesmith177 Yes the comment should be removed. In @akonradi's earlier diff's on this PR he had removed those 4 lines.

**Comment by rodaine** [comment]
https://github.com/lyft/protoc-gen-validate/pull/104 has been merged

**Pull request merged**